### PR TITLE
Partial screening modification

### PR DESCRIPTION
**Pull request by jjwalkowiak**
Added new model for partial screening
Separated partial screening from stopping power
Added runaway rate for Avalanche generation

### CODE REVIEW

File: src/Equations/PartialScreeningFactorWalkowiak.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#include<iostream>
```
Comment (hoppe93) [inline src/Equations/PartialScreeningFactorWalkowiak.cpp:1]: I guess that these functions are described in a paper somewhere (perhaps https://doi.org/10.1063/5.0075859 ?) Maybe you could add a comment at the top of this which just briefly describes what is done and refers to the relevant paper?

**Review by hoppe93 — CHANGES_REQUESTED** [review]
Looks good! If you could just add a comment at the top of ``PartialScreeningFactorWalkowiak.cpp`` which refers to a relevant paper, I think this should be ready to merge! Some of the automatic tests on GitHub utilize the partial screening functionality, suggesting that the old partial screening calculations work as expected.

File: src/Equations/PartialScreeningFactorWalkowiak.cpp (hunk 1 above)
Comment (jjwalkowiak) [reply src/Equations/PartialScreeningFactorWalkowiak.cpp:1]: Done

**Review by hoppe93 — APPROVED** [review]
Now all looks very good! :)

**Pull request merged**